tests/test_normalization.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import os
+from tests import MODEL_DIR, INPUT_DIR, OUTPUT_DIR
+
+from sample_annotator.ontology.package_checklist_normalizer import normalize_package
+from sample_annotator.report_model import AnnotationReport
+from sample_annotator.ontology.triad_normalizer import normalize_triad_slot
+
+"""Test the module can be imported."""
+
+import unittest
+
+
+class TestNormalizastions(unittest.TestCase):
+    """annotation test."""
+
+    def test_env_package_success(self):
```

Comment:
perfect!

consider also doing a normalize_checklist method too while we are at this

or one method that returns a tuple?

Reply:
I think of the checklist as "MIMS" or "MIMAG" etc. Is that the additional normalization you want implemented?

Or a wrapper that takes three inputs and calls `normalize_triad_slot` three times?

Reply:
Even `scoped-mapping` doesn't really perform normalization for triads using two of the slots as context for interpreting the third. As of now, I have implemented even less scoping in `sample-annotator`

Reply:
> I think of the checklist as "MIMS" or "MIMAG" etc
correct